mock-plugin/src/test/java/io/mosip/esignet/plugin/mock/service/MockVCIssuancePluginTest.java

```diff
@@ -0,0 +1,204 @@
+    }
+
+
+    @Test
+    public void getVerifiableCredentialWithLinkedDataProof_withSecureIndividualId_thenPass() throws VCIExchangeException, NoSuchAlgorithmException {
+
+        ReflectionTestUtils.setField(mockVCIssuancePlugin, "verificationMethod", "http://localhost:8080/v1/esignet.mosip.net/oauth/.well-known/jwks.json");
+        List<String> vcCredentialContexts = new ArrayList<>();
+        vcCredentialContexts.add("https://www.w3.org/2018/credentials/v1");
+        ReflectionTestUtils.setField(mockVCIssuancePlugin, "vcCredentialContexts", vcCredentialContexts);
+        ReflectionTestUtils.setField(mockVCIssuancePlugin, "secureIndividualId", true);
+        ReflectionTestUtils.setField(mockVCIssuancePlugin, "storeIndividualId", true);
+        ReflectionTestUtils.setField(mockVCIssuancePlugin, "aesECBTransformation", "AES/ECB/PKCS5Padding");
+        ReflectionTestUtils.setField(mockVCIssuancePlugin,"cacheSecretKeyRefId","cacheSecretKeyRefId");
+        ReflectionTestUtils.setField(mockVCIssuancePlugin,"getIdentityUrl","http://localhost:8080/v1/esignet.mosip.net");
+
+        KeyGenerator generator = KeyGenerator.getInstance("AES");
+        generator.init(256);
+        SecretKey key = generator.generateKey();
+        String individualId = encryptIndividualId("individual-id",key);
+
+        OIDCTransaction oidcTransaction = new OIDCTransaction();
+        oidcTransaction.setIndividualId(individualId);
+        oidcTransaction.setKycToken("kycToken");
+        oidcTransaction.setAuthTransactionId("authTransactionId");
+        oidcTransaction.setRelyingPartyId("relyingPartyId");
+
+        Mockito.when(parsedAccessToken.getAccessTokenHash()).thenReturn("123456789");
+
+        Mockito.when(cache.get("123456789", OIDCTransaction.class)).thenReturn(oidcTransaction);
+        Mockito.when(cacheManager.getCache(Mockito.anyString())).thenReturn(cache);
+
+
+        Map<String, List<KeyAlias>> keyaliasesMap = new HashMap<>();
+        KeyAlias keyAlias = new KeyAlias();
+        keyAlias.setAlias("test");
+        keyaliasesMap.put("currentKeyAlias", Arrays.asList(keyAlias));
+        Mockito.when(dbHelper.getKeyAliases(Mockito.anyString(), Mockito.anyString(), Mockito.any(LocalDateTime.class))).thenReturn(keyaliasesMap);
+
+        JWTSignatureResponseDto jwtSignatureResponseDto = new JWTSignatureResponseDto();
+        jwtSignatureResponseDto.setTimestamp(LocalDateTime.now());
+        jwtSignatureResponseDto.setJwtSignedData("test");
+        Mockito.when(signatureService.jwtSign(Mockito.any())).thenReturn(jwtSignatureResponseDto);
+
+
+        Mockito.when(keyStore.getSymmetricKey(Mockito.any())).thenReturn(key,key);
+
+
+
+        mockVCIssuancePlugin.getVerifiableCredentialWithLinkedDataProof(new VCRequestDto(), "test", new HashMap<>());
+    }
+
+    @Test
+    public void getVerifiableCredentialWithLinkedDataProof_withSecureIndividualId_thenFail() throws VCIExchangeException, NoSuchAlgorithmException {
```

Comment:
What is the difference b/w test case in line 91 and 140

Reply:
changed the name of the method , here the execption is getting througn because of Invalid Alias but the parrent method is just logging that exeption